tests/unix/extra_coverage.py

```diff
@@ -78,3 +78,11 @@
 # test for MP_QSTR_NULL regression
 from frzqstr import returns_NULL
 print(returns_NULL())
+
+# test micropython.heap_locked()
```

Comment:
please put this in a new, separate test file, eg `tests/micropython/heap_locked.py`, with a test at the start to skip it, like:
```python
import micropython
if not hasattr(micropython, "heap_locked"):
    print("SKIP")
    raise SystemExit
```